apps/docs/content/guides/realtime/subscribing-to-database-changes.mdx

```diff
@@ -1,27 +1,101 @@
 
-You can listen to database changes using the [Postgres Changes](/docs/guides/realtime/postgres-changes) extension.
-The following video shows how you can enable this feature for your tables.
+1. [Broadcast](/docs/guides/realtime/broadcast). This is the recommended method for scalability and security.
+1. [Postgres Changes](/docs/guides/realtime/postgres-changes). This is a simpler method that requires no setup on your part, although generally less-scalable.
 
-## Demo
+## Using Broadcast
+
+To automatically send messages when a record is created, updated, or deleted, we can attach a [Postgres trigger](/docs/guides/database/postgres/triggers) to any table. Supabase Realtime provides a `realtime.broadcast_changes()` function which we can use in conjunction with a trigger.
+
+### Broadcast Authorization
+
+[Realtime Authorization](/docs/guides/realtime/authorization) is required for receiving Broadcast messages. This is an example of a policy that allows authenticated users to listen to messages from topics:
+
+{/* prettier-ignore */}
+```sql
+create policy "Authenticated users can receive broadcasts"
+on "realtime"."messages"
+for select
+to authenticated
+using ( true );
+```
+
+### Create a trigger function
+
+Let's create a function that we can call any time a record is created, updated, or deleted. This function will make use of some of Postgres's native [trigger variables](https://www.postgresql.org/docs/current/plpgsql-trigger.html#PLPGSQL-DML-TRIGGER). For this example, we want to have a topic with the name `topic:<record id>` to which we're going to broadcast events.
+
+{/* prettier-ignore */}
+```sql
+create or replace public.your_table_changes() 
+returns trigger 
+language plpgsql
+as $$
+begin
+  perform realtime.broadcast_changes(
+    'topic:' || NEW.id::text, -- topic - the topic to which we're broadcasting
+    TG_OP,                    -- event - the event that triggered the function
+    TG_OP,                    -- operation - the operation that triggered the function
+    TG_TABLE_NAME,            -- table - the table that caused the trigger
+    TG_TABLE_SCHEMA,          -- schema - the schema of the table that caused the trigger
+    NEW,                      -- new record - the record after the change
+    OLD                       -- old record - the record before the change
+  );
+  return null;
+end;
+$$;
+```
+
+### Create a trigger
+
+Let's set up a trigger so the function is executed after any changes to the table.
+
+{/* prettier-ignore */}
+```sql
+create trigger handle_your_table_changes
+after insert or update or delete 
+on public.your_table
+for each row
+execute function your_table_changes ();
+```
+
+#### Listening on client side
+
+Finally, client side will requires to be set up to listen to the topic `topic:<record id>` to receive the events. Do not forget that you will need `private` set to true in this scenario as we are using `realtime.broadcast_changes` to send the messages.
```

Comment:
```suggestion
Finally, on the client side, listen to the topic `topic:<record_id>` to receive the events. Remember to set the channel as a private channel, since `realtime.broadcast_changes` uses Realtime Authorization.
```